### PR TITLE
test on Mafft

### PR DESCRIPTION
**Pull request by RNAer**
This is to test Travis failure of #27 .  This solved the issue of `conda command not found`. It seems Travis made some change so that our original install.sh does not work wrt the `PATH`. The changes made here should be more robust.

### CODE REVIEW

**Comment by RNAer** [comment]
@Jorge-C , would you mind taking a look? @ekopylova , I included your commits/code in this PR. do you think it is ok to merge this one and close your original one?

File: ci/travis/install.sh [hunk 1]
```diff
@@ -12,9 +12,12 @@ esac
...
 wget "${MINICONDA_URL}/${MINICONDA_FILE}"
-bash $MINICONDA_FILE -b
+bash $MINICONDA_FILE -b -p $HOME/miniconda
 
 export PATH=$HOME/miniconda/bin:$PATH
-
-conda update --yes conda
-conda install --yes pip conda-build anaconda-client
+hash -r
+conda config --set always_yes yes --set changeps1 no
```
Comment (Jorge-C) [inline ci/travis/install.sh:19]: Is disabling command line change necessary?

**Comment by Jorge-C** [comment]
Looks good. Does multithreading need to be disabled b/c of a mafft build process limitation? I added another minor question inline.

File: ci/travis/install.sh (hunk 1 above)
Comment (RNAer) [inline ci/travis/install.sh:19]: not tested that individually, but recommended by conda website.

Comment (Jorge-C) [inline ci/travis/install.sh:19]: :+1:

2015-11-04 19:56 GMT-07:00 Zech Xu notifications@github.com:

**Comment by ekopylova** [comment]
Yes, multithreading on Mac OS X needs to be disabled manually by editing the Makefile (instructions given in the readme file for mafft-7.245-with-extensions-src.tgz source, see [here](http://mafft.cbrc.jp/alignment/software/source.html)).

**Comment by Jorge-C** [comment]
Thanks for the quick feedback! Merged.

**Pull request merged**